README.md

```diff
@@ -221,14 +224,25 @@ These variables are set in the same level of the `logging_inputs`, `logging_outp
                      /etc/pki/tls/certs/cert.pem for cert
                      /etc/pki/tls/private/key.pem for key
 ``` 
+Security parameters
 - `logging_pki_authmode`: Specify the default network driver authentication mode. `x509/name` or `anon` are available: Default to "x509/name".
 - `logging_domain`: The default DNS domain used to accept remote incoming logs from remote hosts. Default to {{ ansible_domain if ansible_domain else ansible_hostname }}
 - `logging_permitted_peers`: List of hostnames, IP addresses or wildcard DNS domains which will be allowed by the `logging` server to connect and send logs over TLS.  Default to ['*.{{ logging_domain }}']
 - `logging_send_permitted_peers`: List of hostnames, IP addresses or wildcard DNS domains which will be verified by the `logging` client and will allow to send logs to the remote server over TLS. Default to '{{ logging_permitted_peers }}'.
+
+Server Performance optimization
+- `logging_tcp_threads`: Input thread count listening on the tcp port. Default to 1.
+- `logging_udp_threads`: Input thread count listening on the udp port. Default to 1.
+- `logging_udp_system_time_interval`: Interval to get the system time. udp only. Default to 2.
+- `logging_udp_batch_size`: Maximum number of udp messages. Default to 32.
+- `logging_server_queue_type`: Type of queue. `FixedArray` is available. Default to `LinkedList`.
+- `logging_server_queue_size`: Size of queue. Default to 50000.
```

Comment:
Is this the size in bytes or the number of records?

Reply:
Yes, I'm adding it.